src/rebuild/srv.c

```diff
@@ -1693,13 +1702,16 @@ rebuild_tgt_status_check(void *arg)
 			iv.riv_toberb_obj_count = rpt->rt_toberb_objs;
 			iv.riv_obj_count = status.obj_count;
 			iv.riv_rec_count = status.rec_count;
+			iv.riv_size = status.size;
 		} else {
 			iv.riv_toberb_obj_count = rpt->rt_toberb_objs -
 						  rpt->rt_reported_toberb_objs;
 			iv.riv_obj_count = status.obj_count -
 					   rpt->rt_reported_obj_cnt;
 			iv.riv_rec_count = status.rec_count -
 					   rpt->rt_reported_rec_cnt;
+			iv.riv_size = status.size -
+					   rpt->rt_reported_size;
```

Comment:
seems need to rebase to latest master as there are a few refining at above L1698 ~ L1700, may need to add below to there:
if (status.size < rpt->rt_reported_size)
status.size = rpt->rt_reported_size;

Reply:
ah, ok